### PR TITLE
Update README and CONTRIBUTING with some submission guidelines 📖

### LINKED ISSUE #4: Decide/Include Submission Guidelines
- [x] What type of tasks/pipelines are appropriate for this catalog?
- [ ] What are the requirements to become an OWNER of this catalog?
- [ ] What are the requirements to become an OWNER of a specific item in the catalog?

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This adds information on OWNERS of the whole project and Tasks.
And also adds some initial OWNERS file :angel: 

/cc @ImJasonH @dlorenc 
/cc @navidshaikh @garethr @chmouel @piyush-garg @khrm (as I added you in some task `OWNERS`)

Closes #4 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,4 +12,46 @@ and the individual `CONTRIBUTING.md` files in each respective project.
...
+1. Be invited (and accept your invite) as a read-only collaborator on
+[the tekton organization](https://github.com/tektoncd). If you need
+sponsors and have contributed to the chart, please reach out to the
+existing maintainers, or if you are having trouble connecting with
+them, please reach out to one of the main [OWNERS](OWNERS) of this
+repository.
+2. an `OWNERS` file needs to be added in the `Task` folder. That
+`OWNERS` file should list the maintainers' GitHub login names for both
+the reviewers and approvers sections.
+
```
Comment (dlorenc) [inline CONTRIBUTING.md]: nit: only one new line.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,4 +12,46 @@ and the individual `CONTRIBUTING.md` files in each respective project.
...
+  registry (gcr.io, docker.io, quay.io, …). A bonus if those images are
+  auto-built.
+* Images should not have any major security vulnerabilities
+* Should follow Kubernetes best practices
+* Provide as much paramater values as possible
+* (Nice to have) : provide versions with and without `PipelineResource`
+
+### Owning and Maintaining a Task
+
+Individual tasks can be maintained by one or more users of
```
Comment (dlorenc) [inline CONTRIBUTING.md]: I would even say "should" be maintained.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -12,4 +12,46 @@ and the individual `CONTRIBUTING.md` files in each respective project.
...
+### Technical requirements
+
+* Must pass the Task validation (aka `kubectl create -f task.yaml`
+  should succeed)
+* Images should be published and maintained on an public image
+  registry (gcr.io, docker.io, quay.io, …). A bonus if those images are
+  auto-built.
+* Images should not have any major security vulnerabilities
+* Should follow Kubernetes best practices
+* Provide as much paramater values as possible
```
Comment (dlorenc) [inline CONTRIBUTING.md]: typo/grammar: Provide as many parameter values as possible

**Comment by garethr** [comment]
Acknowledging receipt and +1. Sounds like a good idea, glad someone was thinking about ownership in a shared repo.

**Comment by dlorenc** [comment]
This looks fine to me, do we want to make sure all the new owners ack this first? Up to you.

/hold

**Comment by dlorenc** [comment]
I added a hold just so it doesn't go straight in until you're ready, feel free to remove whenever.

**Comment by vdemeester** [comment]
ping @piyush-garg and @khrm offline, we're all good :angel: 
/hold cancel

**Pull request merged**